kernel/patches-4.14.x/README.md

```diff
@@ -0,0 +1,42 @@
+# How to build 4.14.x based custom Linux kernel for LCOW
+
+You can download the Linux 4.14 source code from kernel.org
```

Comment:
4.14 is not out yet (no branch or tag) so you can't download it...

Reply:
I was basing on the most recently commit on the Linux mainline.  I will remove this 4.14 entry and wait for the source code to be available-- in the meanwhile i could get some more testing done...